luatest/server.lua

```diff
@@ -537,8 +547,7 @@ function Server:exec(fn, args, options)
         )
         error(err, 2)
     end
-
-    return exec_tail(self.net_box:eval([[
+    return exec_tail(self:eval([[
```

Comment:
Please leave empty line as it was before.

Reply:
Yep, added